web-client/src/views/Messages/MessageColumns.test.ts

```diff
@@ -0,0 +1,31 @@
+import {
```

Comment:
This is the "here are the available fields for a messages table" class. Eventually, filters should go here too ... but one step at a time.

My hope is that we can pull out and reuse the message-independent stuff in order to standardize our sortable tables.